python-sdk/src/langwatch/monitors.py

```
@@ -0,0 +1,118 @@
+from langwatch.state import get_instance
+
+
+def _raise_for_status(response: httpx.Response, *, operation: str = "") -> None:
+    if response.is_success:
+        return
+    status = response.status_code
+    detail = ""
+    try:
+        body = response.json()
+        detail = body.get("message") or body.get("error") or ""
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)